test/catRoutes.test.js

```diff
@@ -199,31 +199,3 @@ describe('POST /cats', function () {
             .expect(400,'Error: Request data is not a valid object. ', done)
     }) 
 })
-
-describe('GET /breeds', function () {
```

Comment:
why do these tests need to be removed? It'd be preferable to keep them in unless there's a good reason

Reply:
Tests for the breed router routes are now in their own folder.